tests/web/rpc/test_timeseries_api.py

```diff
@@ -27,7 +115,25 @@ def test_basic(self) -> None:
         assert response.status_code == 200
 
         # STUB response test
-        pbuf_response = AggregateBucket_pb2.AggregateBucketResponse()
-        pbuf_response.ParseFromString(response.data)
+        # pbuf_response = AggregateBucket_pb2.AggregateBucketResponse()
```

Comment:
nit: This looks like something you wanted to add but later on dropped. Should this be commented out or deleted?